api/PclusterApiHandler.py

```diff
@@ -441,6 +441,23 @@ def list_users():
     except Exception as e:
         return {"exception": str(e)}
 
+def delete_user():
+    try:
+        cognito = boto3.client("cognito-idp")
+        username = request.json.get("Username")
+        cognito.admin_delete_user(UserPoolId=USER_POOL_ID, Username=username)
+        return {'Username': username}
+    except Exception as e:
+        return {"exception": str(e)}
```

Comment:
Most other errors are returning a `message` and this should return a tuple with a `500` as the second argument to indicate internal error.

Reply:
Brilliant! Made that change and now you get a nice error message

![image](https://user-images.githubusercontent.com/5545980/156075513-27dd85cd-eadb-47db-a79f-59bf7407b9ec.png)